packages/e2e/package.json

```diff
@@ -100,7 +103,6 @@
     "@types/bunyan": "^1.8.8",
     "@types/jest": "^28.1.2",
     "@types/lodash": "^4.14.182",
-    "@types/node": "14.18.12",
```

Comment:
are these being picked up by the root package.json?

Reply:
Yes, but I now may need to revert this change and just duplicate it in every package, as I think there's a dependency nominating it as a peer dep. Will also look into the CI failure, which isn't failing locally

Reply:
>  I think there's a dependency nominating it as a peer dep

I found a solution to this: https://github.com/input-output-hk/cardano-js-sdk/pull/499/commits/7e02986d4156431b4d63ee2748d10863d1482359